libs/ts/onchain-interactions/src/scripts/cost-calculations.ts

```diff
@@ -55,7 +50,7 @@ const calculateGasCosts = (
   let totalGasUsed = BigInt(0);
 
   for (const tx of transactions) {
-    const gasUsed = BigInt(tx.gasUsed ?? tx.gas_used ?? tx.gas);
+    const gasUsed = BigInt(tx.gasUsed ?? tx.gas_used ?? tx.gas ?? tx.gasused);
```

Comment:
What is `gasused`

`tx` is of type `Transaction` https://github.com/blocksense-network/blocksense/blob/14ad93e4caed2e64347b183db5862ce3307fa582/libs/ts/onchain-interactions/src/types.ts#L104

There is no `gasused` field

Reply:
Forgot to commit the update to the Transaction type

Reply:
4 - the `gas: string` is actually also a gas used field.
This is because different networks name that field in their transactions differently